flask_ipywidgets/example2.py

```diff
@@ -22,7 +22,4 @@ def index():
 
 
 if __name__ == "__main__":
-    from gevent import pywsgi
-    from geventwebsocket.handler import WebSocketHandler
-    server = pywsgi.WSGIServer(('', 5000), app, handler_class=WebSocketHandler)
-    server.serve_forever()
+    flask_ipywidgets.main_factory(app)()
```

Comment:
Trailing newlines are not strictly required by [PEP-8](https://www.python.org/dev/peps/pep-0008/#blank-lines), but are recommended.

Ditto re `.travis.yaml` above.

Reply:
Thx @rs2 will follow up shortly.